### PR TITLE
make windcave payment flow a bit clearer

### PR DESCRIPTION
**Pull request by djpnewton**

### CODE REVIEW

File: lib/deposit.dart [hunk 1]
```diff
@@ -657,26 +661,25 @@ class _FiatDepositDetailScreenState extends State<FiatDepositDetailScreen> {
...
+                      ZapOnPrimary,
+                      ZapPrimary,
+                      ZapPrimaryGradient,
+                      'Complete Payment',
+                      width: ButtonWidth,
+                      height: ButtonHeight)
+                ])
+              : SizedBox(),
           VerticalSpacer(),
-          Row(mainAxisAlignment: MainAxisAlignment.center, children: <Widget>[
```
Comment (4855062503) [inline lib/deposit.dart:679]: Any benefit to removing the typecasting?

File: lib/deposit.dart [hunk 2]
```diff
@@ -657,26 +661,25 @@ class _FiatDepositDetailScreenState extends State<FiatDepositDetailScreen> {
...
-                      size: 200.0,
-                    ),
-                    Text('${_deposit.paymentUrl}')
-                  ]),
-                  onTap: () => urlLaunch(_deposit.paymentUrl))
-              : SizedBox(),
           ListTile(
               title: Text('Status'),
               subtitle: Text('${_deposit.status.toUpperCase()}')),
+          _deposit.paymentUrl != null &&
```
Comment (4855062503) [inline lib/deposit.dart:667]: Way nicer solution.

**Review by 4855062503 — APPROVED** [review]
LGTM.

File: lib/deposit.dart (hunk 1 above)
Comment (djpnewton) [reply lib/deposit.dart:679]: its not required, it is inferred by the compiler

**Pull request merged**